app/common/ServiceAccountTypes.ts

```diff
@@ -22,6 +23,7 @@ export interface ServiceAccountApiResponse {
 export interface ServiceAccountCreationResponse extends ServiceAccountApiResponse {
   id: number;
   key: string;
+  login: string|undefined;
```

Comment:
Isn't it supposed to be mandatory in the response?

If so and if my comment for `ServiceAccountApiResponse` is true, you may remove this property (which is inherited from this interface)